README.md

```diff
@@ -1,38 +1,31 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+LeaderBoard
+
 
-# LeaderBoard
 In this activity I will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I should develop a first working version of the app following a wireframe, but without styling - just focus on functionality. In following activities, I will consume the Leaderboard API using JavaScript async and await and add some styling.
+Built With
 
-# Built With
+html 5 , css
+visual studio code
 
-    html 5 , css
-    visual studio code
-    
 clone the project: https://github.com/Khanetmoi/leader.git
+👤 Author1
 
-# 👤 Author1
-
-    GitHub: @Khanetmoi
-    Twitter: @InayatVan
-    LinkedIn: Khan Bitsindou
-
+GitHub: @Khanetmoi
+Twitter: @InayatVan
+LinkedIn: Khan Bitsindou
 
-
-# 🤝 Contributing
+🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the issues page.
-Show your support
+Feel free to check the issues page. Show your support
 
-Give a ⭐️ if you like this project!
-Acknowledgments
+Give a ⭐️ if you like this project! Acknowledgments
 
-    Hat tip to anyone whose code was used
-    Inspiration
-   
+Hat tip to anyone whose code was used
+Inspiration
 
-📝 License
 
-This project is [MIT](./LICENSE) licensed.
+📝 License
 
+This project is MIT licensed.
```

Comment:
- [ ] Kindly make sure your README looks similar to the required **[Microverse template](https://github.com/microverseinc/readme-template)** with correct titles on specific sections, social links on your user name,... Right now your README looks like a short paragraph without titles on different sections and other details. I can suggest you take a look at this **[Template](https://github.com/tresorsawasawa/Leaderboard_App/blob/development/README.md)** and use it as an example to complete your README.